packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -98,6 +100,33 @@ const Text = styled.div`
     position: relative;
 `;
 
+const TextHover = styled.div`
+    position: relative;
+    &:after {
+        content: '';
+        position: absolute;
+        top: -4px;
+        left: -4px;
+        right: -4px;
+        bottom: -4px;
+        border-radius: 8px;
+        transition: all 150ms ease-out;
```

Comment:
Pls constult transition style/length with our designers + it's probably a good idea to unify the transitions overally and put them in a config/theme.

Reply:
done

Reply:
I can confirm that this is a correct transition which our designer wanted.